deployment/templates/fluentd.yaml

```diff
@@ -58,20 +44,15 @@ objects:
             component: "fluentd"
         spec:
           serviceAccountName: aggregated-logging-fluentd
+          nodeSelector:
+            logging-infra: fluentd
           containers:
           - name: fluentd-elasticsearch
-            image: logging-fluentd
+            image: ${IMAGE_PREFIX}logging-fluentd
```

Comment:
Is this necessary since we should be consuming an image from an imagestream which would abstract the location?